helm-chart/enroute/templates/roles.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "enroute.fullname" . }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+      - endpoints
+      - nodes
+      - pods
+      - secrets
+    verbs:
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - get
+  - apiGroups:
+      - ""
+    resources:
+      - services
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - networking.k8s.io
+    resources:
+      - ingresses
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+    - "enroute.saaras.io"
+    resources:
+      - ingresses
+      - gatewayhosts
+      - globalconfigs
+      - httpfilters
+      - routefilters
+      - tlscertificatedelegations
+    verbs:
+      - get
+      - list
+      - watch
+      - put
+      - post
+      - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "enroute.fullname" . }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ include "enroute.fullname" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "enroute.fullname" . }}
+    namespace: {{ .Release.Namespace }}
```

Comment:
We use the namespace here to reference the ```ServiceAccount``` but the creation of ```ServiceAccount``` doesn't have the namespace in ```serviceaccount.yaml```

Reply:
See above